### ISSUE TITLE
vendor.js missing in build

### ISSUE THREAD

**Issue by larzknoke** [betrueagency/nx-reactjs-elementor#1]
Hi, 

nice project by the way. It's working fine so far. The only problem after building and install the plugin I have a console error:

`http://reactwp2.local/wp-content/plugins/my-project/dist/vendor.js?ver=1.0.0 net::ERR_ABORTED 404 (Not Found)`

<img width="1265" alt="Bildschirmfoto 2022-01-28 um 11 23 16" src="https://user-images.githubusercontent.com/6797960/151531231-bbdd4c61-feb2-4852-951a-4005824404ae.png">

on the dev server the vendor is included:

<img width="1086" alt="Bildschirmfoto 2022-01-28 um 11 23 19" src="https://user-images.githubusercontent.com/6797960/151531305-b60f588f-eb11-4658-a351-85cd1e1770c8.png">

**Comment by ilpanzi** [comment]
Same problem here. 
I've just imported the zipped file in my test website and this appear on my console:
![image](https://user-images.githubusercontent.com/26737813/152334490-0befd8c4-024a-421c-a5ec-e6ea2ff295dd.png)

**Comment by alizarion** [comment]
@larzknoke @ilpanzi  sorry for the late answer, I solved the problem in the release 0.0.13 
it was due to an evolution of a build options on @nwrl/web that changed the final name of the production files without possibility to change this behavior.

I introduced a new parameter on the plugin builder allowing to rename dynamically build files and face this kind of issue.
on a new project, the plugin will generate the files with the right parameters juste follow the readme.

on an existing project add the parameter configuration of the target build `"outputHashing": "none",`
```json
  "targets": {
    "build": { 
      ...
      "configurations": {
          ...
          "outputHashing": "none",
           ...
        }
      }
    },
```
and replaceFilePattern:.esm.js

```json
"plugin": {
      "executor": "@betrue/react-elementor:build",
      "options": {
        "plugin": "my-project",
        "replaceFilePattern": ".esm.js"
      }
    },
```
